python/mujoco/rollout.cc

```diff
@@ -230,9 +280,24 @@ PYBIND11_MODULE(_rollout, pymodule) {
           py::gil_scoped_release no_gil;
 
           // call unsafe rollout function
-          InterceptMjErrors(_unsafe_rollout)(
-              model_ptrs, data, nroll, nstep, control_spec, state0_ptr,
-              warmstart0_ptr, control_ptr, state_ptr, sensordata_ptr);
+          if (nthread > 1 && nroll > 1) {
+            int chunk_size_final = 1;
+            if (!chunk_size.has_value()) {
+              chunk_size_final = std::max(1, nroll / (10 * nthread));
+            }
+            else {
```

Comment:
The else on a new line thing is something that we do in C code, and only to facilitate putting a comment above the block. Let's attach all else's `} else {` in C++ code as in the style guide.